src/ActiveLogin.Authentication.BankId.AspNetCore.Qr/BankIdAuthenticationBuilderQrCodeExtensions.cs

```diff
@@ -0,0 +1,17 @@
+using ActiveLogin.Authentication.BankId.AspNetCore.Models;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+
+namespace ActiveLogin.Authentication.BankId.AspNetCore.Qr
+{
+    public static class BankIdAuthenticationBuilderQrCodeExtensions
+    {
+        public static IBankIdAuthenticationBuilder UseDefaultQrCodeGenerator(this IBankIdAuthenticationBuilder builder)
```

Comment:
Rename to:
`UseQrCoderQrCodeGenerator`

I think we should have the name of the lib in the name, if we add support for other libs.